src/v/redpanda/application.h

```diff
@@ -84,6 +86,11 @@ class application {
         return cfg.developer_mode() && cfg.enable_coproc();
     }
 
+    bool proxy_enabled() {
+        const auto& cfg = config::shard_local_cfg();
+        return cfg.developer_mode() && cfg.enable_proxy();
```

Comment:
we don't need the developer_mode() here, we just have to clearly mark that in the documentation.

Reply:
Fixed